common/djangoapps/student/models.py

```diff
@@ -61,10 +61,21 @@
 
 
 UNENROLL_DONE = Signal(providing_args=["course_enrollment", "skip_refund"])
+ENROLL_STATUS_CHANGE = Signal(providing_args=["event", "user", "course_id", "mode", "cost", "currency"])
 log = logging.getLogger(__name__)
 AUDIT_LOG = logging.getLogger("audit")
 SessionStore = import_module(settings.SESSION_ENGINE).SessionStore  # pylint: disable=invalid-name
 
+# enroll status changed events - signaled to email_marketing.  See email_marketing.tasks for more info
+
+# ENROLL signal used for free enrollment only
+ENROLL_STATUS_CHANGE_ENROLL = 'enroll'
+ENROLL_STATUS_CHANGE_UNENROLL = 'unenroll'
+ENROLL_STATUS_CHANGE_UPGRADE_ADD_CART = 'upgrade_start'
+ENROLL_STATUS_CHANGE_UPGRADE_COMPLETE = 'upgrade_complete'
+ENROLL_STATUS_CHANGE_PAID_COURSE_ADD_CART = 'paid_start'
+ENROLL_STATUS_CHANGE_PAID_COURSE_COMPLETE = 'paid_complete'
```

Comment:
Nit: In other places, we've done enums like:

    class EnrollStatusChange(object):
        enroll = 'enroll'
        ...